packages/plugin-e2e/src/e2e-selectors/types.ts

```diff
@@ -196,9 +196,23 @@ export type Pages = {
             generalTypeSelectV2: string;
             previewOfValuesOption: string;
             submitButton: string;
+            generalNameInputV2: string;
+            applyButton: string;
           };
         };
       };
+      Close: string;
+    };
+    TemplateVariables: {
+      submenuItemLabels: (item: string) => string;
+      submenuItemValueDropDownValueLinkTexts: (item: string) => string;
+      submenuItemValueDropDownDropDown: string;
+      submenuItemValueDropDownOptionTexts: (item: string) => string;
+    };
```

Comment:
Same for these - already defined in a different [place](https://github.com/grafana/grafana/blob/main/packages/grafana-e2e-selectors/src/selectors/pages.ts/#L77-L85).

Reply:
Same here